crates/resvg/src/image.rs

```diff
@@ -87,7 +87,22 @@ mod raster_images {
 
         let options = DecoderOptions::default().jpeg_set_out_colorspace(ColorSpace::RGBA);
         let mut decoder = zune_jpeg::JpegDecoder::new_with_options(data, options);
-        let img_data = decoder.decode().ok()?;
+        decoder.decode_headers().ok()?;
+        let output_cs = decoder.get_output_colorspace()?;
+
+        let img_data = {
+            let data = decoder.decode().ok()?;
+            match output_cs {
+                ColorSpace::RGBA => data,
+                // For some reason zune jpeg won't coerce luma JPEGs into RGBA, so we do it manually.
+                ColorSpace::Luma => data
+                    .into_iter()
+                    .flat_map(|p| [p, p, p, 255])
+                    .collect::<Vec<_>>(),
+                _ => return None,
```

Comment:
Would it make sense to warn here? (Do we even have warning infrastructure available?)

Reply:
We do, but we already print a warning here in case JPEG decoding fails (i.e. the function returns `None`): https://github.com/linebender/resvg/blob/a53573b7f49a48ea73e61d92c40f77df96267a4a/crates/resvg/src/image.rs#L65

So I think this should be enough. :)